shared/src/vyxal/Context.scala

```diff
@@ -57,6 +57,7 @@ class Context private (
     */
   def pop(): VAny =
     if useStack && parent.isDefined then return parent.getOrElse(this).pop()
+    stack.filterInPlace(!_.isInstanceOf[VNull])
```

Comment:
Why do this when peeking and popping? Why not only filter out `VNull`s when pushing?